var/spack/repos/builtin/packages/dbcsr/package.py

```diff
@@ -95,6 +109,15 @@ def cmake_args(self):
 
             args += ['-DWITH_GPU=%s' % gpuver]
 
+        if self.spec.satisfies('+rocm'):
+            amd_arch = self.spec.variants['amdgpu_target'].value[0]
+
+            gpuver = {
+                'gfx906': 'Mi50'
+            }[amd_arch]
+
+            args.append('-DWITH_GPU={0}'.format(gpuver))
```

Comment:
```suggestion
            args += ['-DWITH_GPU={0}'.format(gpuver)]
```
... just to be consistent with the rest